pyproject.toml

```diff
@@ -90,6 +90,7 @@ readme-renderer = "^24.0"
 requests-mock = "^1.9.3"
 tox = "^3.23.0"
 troposphere = "^4"
+ipdb = "^0.13.13"
```

Comment:
what is this for?

Reply:
It's the debugger and I've added it to the dev dependies section. It was quite a bit of work to install a debugger so seems like it probably is of benefit to stay here? I can't think of any reason not to?

Reply:
My view on dependencies are that they should only be added if used otherwise it should not be there.  Your thoughts @jfalkenstein ?

Reply:
Yeah, I agree. There isn't a good reason to put this into our dependencies. Btw, you can always still `pip install` using poetry in your own environment if you need to throw in an additional thing into your virtual environment. You can either run `poetry run pip install ipdb` or run `poetry shell` then within the shell run `pip install ipdb`. It doesn't need to be in our pyproject.toml (and I don't think it should be).

Reply:
Ok removed.